### ISSUE TITLE
require.context is not a function

### ISSUE THREAD

**Issue by ronderksen** [storybooks/storyshots#19]
In my Storybook config, I use the following code to gather all the stories:

```
const req = require.context('../../src/public', true, /.story.js$/)

function loadStories() {
  req.keys().forEach((filename) => req(filename))
}
```

Which is documented here: https://getstorybook.io/docs/basics/writing-stories

However, when I run storyshots, I get an error saying that require.context is not a function.

**Comment by arunoda** [comment]
Yes. That's true. 
We need to add support for that. 
@roonyh check we can use something like this: https://www.npmjs.com/package/enhanced-require inside node.

**Comment by trevordmiller** [comment]
I'm using the config directly from the documentation and getting this error when running `storyshots` directly without any modifications:

# Documentation

https://getstorybook.io/docs/basics/writing-stories)
![screen shot 2016-09-29 at 2 49 35 pm](https://cloud.githubusercontent.com/assets/5497885/18971878/02b1782a-8654-11e6-9319-c725265857b2.png)

# .storybook/config.js

![image](https://cloud.githubusercontent.com/assets/5497885/18971905/2062653c-8654-11e6-82d6-cc5dcd41852f.png)

# story placement next to components

![screen shot 2016-09-29 at 2 51 41 pm](https://cloud.githubusercontent.com/assets/5497885/18971928/4697238c-8654-11e6-9062-9791c571a5c1.png)

# npm script for running `storyshots`

![image](https://cloud.githubusercontent.com/assets/5497885/18971949/5d6a99ea-8654-11e6-8641-9e18df9f1aa5.png)

# error when run

![image](https://cloud.githubusercontent.com/assets/5497885/18972023/b48694c2-8654-11e6-8fa0-378990718c73.png)

**Comment by arunoda** [comment]
@trevordmiller Yep. That's an issue becuse now we are running stories in NodeJS. It don't have require.context. 

We have a fix for that and landing to storyshots soon: https://github.com/kadirahq/storyshots/pull/30

**Comment by roonyh** [comment]
@ronderksen @trevordmiller, Released version 1.1.2 with #30. It should be able to support using`require.context` inside storybook's main config file(`.storybook/config.js`).

**Comment by arunoda** [comment]
Guys, we had a [regression](https://github.com/kadirahq/storyshots/pull/30#issuecomment-250805615) for this issue and we revert `v1.1.2`.

We were looking at some [alternate solution](https://github.com/kadirahq/storyshots/pull/30#issuecomment-250814311) and it'll work great.

**Comment by usulpro** [comment]
still have this issue. 
Maybe there is a way to exclude some files from storyshorts?

**Comment by roonyh** [comment]
This is fixed in #67
